docs/VL53Lxx.md

```diff
@@ -58,51 +57,51 @@ Sample boards for VL53L0X. Similar boards exist for VL53L1X.
+If your Tasmota binary includes the drivers for the TSL2561 and/or the TSL2591 they must disabled. Similarly, the driver for the other VL53LXX device must also be disabled. Here are the commands to type in the console for each sensor:
 
 * To use a VL53L0X : `backlog i2cdriver16 0;i2cdriver40 0;i2cdriver31 1;i2cdriver54 0`
 * To use a VL53L1X : `backlog i2cdriver16 0;i2cdriver40 0;i2cdriver31 0;i2cdriver54 1`
 
 
-After a reboot Tasmota will detect the VL53LXX automatically is the proper driver has been enabled.
+After configuring these optinos, Tasmota will reboot and detect the correct VL53LXX device automatically.
 
 ![image](_media/peripherals/vl53l0x.png)
 
-Sensor sends a  `tele/%topic%/SENSOR` JSON response:
+These sensors send a `tele/%topic%/SENSOR` JSON response:
 
-For VL53L0X:
+For the VL53L0X:
 ```json
 {"Time":"2019-12-20T11:29:22","VL53L0X":{"Distance":263}}
 ```
-Or for VL53L1X
+For the VL53L1X
 ```json
 {"Time":"2019-12-20T11:29:22","VL53L1X":{"Distance":263}}
 ```
 
 ## Configuration for Multiple VL53LXX sensors in parallel
 
-Tasmota supports by default up to 8 of these sensors in parallel. Note that they all must be of the same model (either all VL53L0X or all VL53L1X). In any case, the above I2C GPIO and I2C driver configuration remain valid.
+Tasmota, by default, supports up to 8 of sensors of the same model in parallel. The I2C GPIO and I2C driver configuration listed above are the same.
 
-When using multiple VL53LXX, it is required to wire the XSHUT pin of each sensors to a dedicated free GPIO and assign as `VL53LXX XSHUT` 1 to 8. This is to let Tasmota change by software the I2C address of those and give them an unique address for operation. The sensor don't save its address and this procedure is performed automatically at every restart. The Addresses used for this are by default 0x78 (120) to 0x7F (127). As for a single sensor, you must not use any other I2C device on those addresses. As the date of writing, there is currently no I2C device supported by Tasmota with those addresses. However such devices exists and may be supported by later versions of Tasmota. You can change the relocation address by a compile option, see the [compile options](#compile-options) section.
+When using multiple VL53LXX devices the XSHUT pin of each sensor must be wired to a dedicated GPIO and assigned as `VL53LXX XSHUT` 1 to 8. This allows Tasmota to programatically change the I2C address of each sensor so that they are unique. The sensors don't save this new address so this procedure is performed automatically at every restart. The addresses used for this are, by default, 0x78 (120) to 0x7F (127). As a concequence, no other devices can use those addresses. As of January 2023, there are no I2C devices supported by Tasmota with those addresses. However, such devices exist and may be supported by later versions. You can change the reassigned addresses via a compile option, see the [compile options](#compile-options) section.
```

Comment:
> concequence

Reply:
Isn't that how it is spelled?  Or am I missing something?

Reply:
No, it's spelled 'consequence'
You could also use 'result'